tests/unit/test_ratmama.py

```diff
@@ -178,6 +206,24 @@ async def test_manual_signal(bot_fx,
     assert rescue.code_red == code_red
 
 
+async def test_manual_signal_service_account(
+        bot_fx,
+        async_callable_fx,
+        monkeypatch):
+
+    context = await Context.from_message(
+        bot_fx,
+        "#unit_test",
+        "unit_test",
+        "Ratsignal some_service")
+
+    monkeypatch.setattr(context, "reply", async_callable_fx)
```

Comment:
[question] why was the reply attribute patched here? I don't see any usage of `async_callable_fx` used in an assertion as a result of this patch?